src/pages/SingleCoursePages.js

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect} from "react";
+import { Routes, Route, Link, NavLink, useNavigate, useParams } from 'react-router-dom'
+import apiUtil from '../utils/apiUtil'
+
+//replcae this with the API call. (Finsished)
+//const course = require('./data/courseData.json') 
+
+//localhost:3000/homepage/course/1
+
+//This function is going to be used to display the single course page from when the
+//user selects the course.  
+function DisplayCoursePage(){
+//using useParams() here to grab the number in the course they select then finding the data
+//associated with that course.
+let { paramID } = useParams()
+
+useEffect(() => {
+    const getCourseData = async () => {
+        const response = await apiUtil('get', `/courses/${paramID}`)
+
+        if (response.status === 200) {
+            return (
```

Comment:
I have a few concerns about this:
1. I understand we are trying to render content in this component about the course, but first we need to set it - i.e. we need to set course to the course which we have gotten in the API response
2. These fields `course_name`, `crn`, and `course_instructor` don't exist on the course object that we are passing back in the API response. I would recommend testing in Postman so you can see what information needs to be displayed.
3. useEffect shouldn't have a return, and definitely not JSX. useEffect can, however, update the state